Release sage 2.0.0 opt/torsion (#419)
openforcefield/qca-dataset-submission: submissions/2025-02-10-OpenFF-Sage-2.0.0-Torsion-Drive-Training-Dataset-v1.0/README.md

# OpenFF Sage 2.0.0 Torsion Drive Training Dataset v1.0

## Description

A quantum chemical (QC) dataset curated to train the OpenFF 2.0.0 Sage torsion 
potentials. This QC dataset with the OpenFF default level of theory, B3LYP-D3BJ/DZVP, 

> is used to benchmark Sage geometries and energetics. These optimized conformer 
> geometries where used to train one dimensional torsional profiles. This Generation 2 

Reviewer: ```suggestion
consists of one dimensional torsional profiles used to train torsion parameters. This Generation 2 
```

dataset increases chemical diversity when compared to Generation 1, which are of 
value to our industry partners. Large molecules (>20 heavy atoms) were also included, 
including more flexible molecules and a greater degree of conformational variation 
which provide intramolecular interactions. This is the complete TorsionDrive dataset 

> used for training OpenFF 2.0.0 Sage, consisting of the following datasets: 

Reviewer: ```suggestion
used for training OpenFF 2.0.0 Sage, consisting of data drawn from the following datasets: 
```

- [OpenFF Gen 2 Torsion Set 3 Pfizer Discrepancy 2](https://github.com/openforcefield/qca-dataset-submission/tree/7f8ed2ab6c8acc4521c8ca45ff4f587b20f0bcda/submissions/2020-03-23-OpenFF-Gen-2-Torsion-Set-3-Pfizer-Discrepancy-2)
- [OpenFF Gen 2 Torsion Set 4 eMolecules Discrepancy 2](https://github.com/openforcefield/qca-dataset-submission/tree/7f8ed2ab6c8acc4521c8ca45ff4f587b20f0bcda/submissions/2020-03-23-OpenFF-Gen-2-Torsion-Set-4-eMolecules-Discrepancy-2)
- [OpenFF Gen 2 Torsion Set 5 Bayer 2](https://github.com/openforcefield/qca-dataset-submission/tree/7f8ed2ab6c8acc4521c8ca45ff4f587b20f0bcda/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-5-Bayer-2)
- [OpenFF Gen 2 Torsion Set 6 supplemental 2](https://github.com/openforcefield/qca-dataset-submission/tree/7f8ed2ab6c8acc4521c8ca45ff4f587b20f0bcda/submissions/2020-03-26-OpenFF-Gen-2-Torsion-Set-6-supplemental-2)

The `HydrogenBondFilter(method='baker-hubbard')` filter was applied, and the 
following record IDs were dropped due to issues with ForceBalance: 6098580, 2703504, 
2703505, 18045478. Further information can be found in the curation scripts for the 
linked repositories.

## General Information

> * Date: 2024-12-17

Reviewer: Update date?
